pages/1.13/cli/command-reference/dcos-marathon/dcos-marathon-about/index.md

```diff
@@ -10,21 +10,103 @@ enterprise: false
 
 
 # Description
-The `dcos marathon about` command will display the `info.json` file for DC/OS Marathon.
```

Comment:
what does the info.json file capture I think we should clarify it is for Marathon configuration!

Reply:
I'm not sure to understand, doesn't `file for DC/OS Marathon.` make it clear that this concerns the Marathon configuration?